src/SwitcherProvider.js

```diff
@@ -26,15 +23,33 @@ export default function SwitcherProvider(props) {
     };
   }, []);
 
+  const providedMethods = useMemo(() => {
+    const getListenerList = type => {
+      switch (type) {
+        case 'load':
+          return loadListeners.current;
+        case 'popstate':
+          return popStateListeners.current;
+        case 'hashchange':
+          return hashChangeListeners.current;
+        default:
+          throw new Error('womp womp');
```

Comment:
nit: lets make this error more helpful even if it technically should never be hit.

Reply:
lol that's not a nit. thats just a good pr comment